server_admin/topics/user-federation/ldap.adoc

```diff
@@ -51,6 +58,13 @@ UNSYNCED::
   It is up to you to figure out how to synchronize back to LDAP. This allows {project_name} deployments to support
   updates of user metadata on a read-only LDAP server.  This option only applies when you are importing users from LDAP into the local {project_name} user database.
 
+NOTE: When LDAP provider is created, the set of initial <<_ldap_mappers,LDAP mappers>> is created. The mappers are configured in a "best-effort" way based on
+      your used LDAP vendor, edit mode and `Import Users` switch. For example in case of UNSYNCED edit mode, the mappers are pre-configured
+      in a way that particular user attribute is preferably read from the DB instead of from the LDAP. However when you later change the edit mode,
+      the mappers configuration will not be changed as it is not easily possible to detect if they were manually changed in the meantime.
+      This means that it is recommended to NOT update edit mode switch, but rather always decide when creating LDAP provider. This applies for
```

Comment:
s/it is recommended to NOT update edit mode switch/it is recommended NOT to update the `Edit Mode` switch/g

Reply:
Thanks. I've changed this sentence to make it more clear based on your suggestion:

This means that it is recommended NOT to update the `Edit Mode` switch, but rather always decide on `Edit Mode` when creating LDAP provider.